src/nvim/ui.coffee

```diff
@@ -92,6 +144,9 @@ class UI extends EventEmitter
           + ',' + Math.floor(e.clientY / @char_height) \
           + '>'
 
+      if config.blink_cursor
+        pauseBlink()
```

Comment:
So that you can see it whenever you're interacting with it. This only happens when the mouse button is down.